### PR TITLE
alsa_play.c add suggestion to use dmix

### PR DESCRIPTION
**Pull request by webstean**

### CODE REVIEW

File: modules/alsa/alsa_play.c [hunk 1]
```diff
@@ -136,7 +136,8 @@ int alsa_play_alloc(struct auplay_st **stp, const struct auplay *ap,
 	if (err < 0) {
 		warning("alsa: could not open auplay device '%s' (%s)\n",
 			st->device, snd_strerror(err));
-		goto out;
+		info("consider using dmix as your default alsa device");
```
Comment (cspiel1) [inline modules/alsa/alsa_play.c:139]: Good idea to add an info if the error occurs. Anyway we should solve this.

Please add a '\n' at end of line!

File: modules/alsa/alsa_play.c [hunk 2]
```diff
@@ -136,7 +136,8 @@ int alsa_play_alloc(struct auplay_st **stp, const struct auplay *ap,
...
+		out;
```
Comment (cspiel1) [inline modules/alsa/alsa_play.c:140]: "out;" ?

Should stay "goto out;".

File: modules/alsa/alsa_play.c (hunk 1 above)
Comment (webstean) [reply modules/alsa/alsa_play.c]: oops - sorry, fixed.

**Comment by alfredh** [comment]
@webstean many thanks for your improvements :)

**Pull request merged**